packages/core/src/features/genomic/ssmsSummary/ssmsSummarySlice.ts

```diff
@@ -62,6 +63,8 @@ const slice = createSlice({
   extraReducers: (builder) => {
     builder
       .addCase(fetchSsms.fulfilled, (state, action) => {
+        if (state.requestId != action.meta.requestId) return state;
+
         const response = action.payload;
 
         state.summaryData = response.data.hits.map((hit) => ({
```

Comment:
state.summaryData is set here.  state.ssms is not.  is state.ssms unused, which makes this change ok.  Or is it set elsewhere, which would require additional logic for preventing race-conditions?